.github/workflows/tests.yml

```diff
@@ -5,7 +5,7 @@ on:
       - main
   pull_request:
 jobs:
-  format:
+  lint:
```

Comment:
Can't we just remove this step and re-use the already existent `lint` step below?